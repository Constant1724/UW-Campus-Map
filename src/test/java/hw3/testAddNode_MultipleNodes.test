# START_TEST
# NAME MultipleNodes
# test if we can add multiuple Node with data to the graph.

# Create a graph
CreateGraph graph1

# Add multiple nodes
AddNode graph1 1
AddNode graph1 1Content
AddNode graph1 Content
# END_TEST