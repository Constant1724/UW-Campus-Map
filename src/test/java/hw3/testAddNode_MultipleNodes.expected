# START_TEST
# NAME MultipleNodes
# test if we can add multiuple Node with data to the graph.

# Create a graph
created graph graph1

# Add multiple nodes
added node 1 to graph1
added node 1Content to graph1
added node Content to graph1
# END_TEST